IMPORTANT.md

```diff
@@ -0,0 +1,13 @@
+1、Python\Lib\site-packages\中的crypt改成Crypto
+2、pip install pycrypto
+3、不行就pip uninstall Crypto 
+        pip uninstall pycrypto
+        pip install Crypto
+        pip install pycrypto
+    然后重复1 2
+4、注释掉core\SharPyShellPrompt.py的33行
+ #signal.signal(signal.SIGTSTP, lambda s, f: self.do_quit())
```

Comment:
if you comment out this line when you are in the terminal and press ctrl+z you will leave traces on the target system.